pkg/systemd/services/services.scss

```diff
@@ -95,3 +88,45 @@
 .pf-m-grid-md.pf-c-table [data-label] {
     display: revert;
 }
+
+// Magic pixie dust to make the list a bit faster and more furious
+.services-list {
+    // Prevent reflows; only applies to desktop (mobile changes display)
+    // table-layout: fixed;
```

Comment:
Let's not keep in comments unused CSS

Reply:
Yep, I mentioned removing this above. :wink: This PR was to save my work off of my computer and show something that is working even if it's in a rough state.

I wanted to re-visit this line of CSS, as it's a huge performance win, but I'll probably just remove it as we're not really using tables as tables here (where everything needs to line up).

In the long run, we should probably use another widget for the lists rather than our own listing widget which uses composable tables.

 (It could be something like a grid (PF Grid even) which has a flex inside (PF Flex even) and that's it. It'd simplify the DOM and layout code quite a lot. But it also would mean an overhaul to the JSX versus a few lines of CSS to override the display.)